### PR TITLE
Lots of update to standardize events page

### LINKED ISSUE #1109: Standardize Events Page - Update tab title
### Overview
Browser tab title is not up to date

### Action Items
- [x] If possible, please change the name of this page on the browser tab from Hack Nights to Events - Hack for LA

### Resources/Instructions
image
#1041

### LINKED ISSUE #1111: Standardize Events Page - Below Header Standardization
### Overview
Below header css needs to be updated

### Action Items
- [x] To stay consistent with other pages (including the Getting Started page), add an additional class to the div that currently has class .getting-started-page , and call this new class .content-section 
- [x] Add another new class to the same div from above^ , this time with classname .events-page 
- [x] Remove .getting-started-page class from the same div mentioned above^ 
- [x] Adjust size and margin of “our Events” (and “Our Locations” below it) titles within the HTML-
- [x] Adjust font size and weight of headings within column cards (“Meetup Events” etc)- should be-
  - [x] ``` font-size: 1.5rem        (24px)
font-weight: 700
line-height: 35px ```

### Resources/Instructions
Figma - https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=6747%3A49

### LINKED ISSUE #1112: Standardize Events Page - Left Column Updates
### Overview
Left column card meetup css needs to updated

### Action Items
- [x] Add more padding in column cards- should be-
```padding: 32px ```
- [x] Change name of  “.getting-started-page-card-hack-nights”
class - should be renamed - 
```.column-card-lg--events```
- [x] Adjust font size and weight of column card subheadings - should be - 
  - [x] ```font-size: 1.25rem    (20px) font-weight: 700 ; margin: 30px ;      ```
- [x] Put subheadings and paragraphs in same div, and align to the right of their  images (see far right for reference)
- [x] 72px margin between between each Meetup category
- [x] 16px margin between subheadings and paragraphs
- [x] 80px margin between last paragraph and Meetup button

### Resources/Instructions
Figma - https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=6754%3A1297
#1041

### LINKED ISSUE #1113: Standardize Events Page - Right Column Updates
### Overview
Right column css and styling needs to be updated

### Action Items
- [x] Add 72px margin between heading and first paragraph (same as in the Meetup column card)
- [x] Line up image and paragraph more evenly
- [x] Onboarding link leads to wrong url- should be to Getting Started page (https://www.hackforla.org/getting-started)
- [x] Make time zone text bold- should be - `font-weight: 700 `
- [x] Make all of the days of week ALL CAPS (MONDAY, TUESDAY, etc. - in the html), bold, and 18px - should be- 
  - [x] ```font-size: 1.125rem  ;font-weight: 700 ;+ in HTML: all-caps titles ```
- [x] Add 30px margin above the days of week and 10px margin below  - `margin: 30px 0 10px 0`
- [x] Remove the padding that pushes the meeting times to the right- Add: `padding: 0`

### Resources/Instructions
Figma - https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md
#1041

### LINKED ISSUE #1114: Standardize Events Page - Covid Banner and Mobile Drop Down Update
### Overview
css needs to updated for covid banner and mobile drop downs

### Action Items
- [x] Adjust banner font size, weight, and margin - should be -
```
font-size: 1rem         (16px)
font-weight: 700
margin: auto 29%
```
- [x] Decrease the padding of the page cards in their collapsed form- should be- `padding: 5px 15px`
- [x] Adjust the border radius- should be - ` border-radius: 20px`

### Resources/Instructions
Figma - https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=6777%3A16
#1041

### LINKED ISSUE #1116: Standardize Events Page - Update right columns date-time display
### Overview
We need to edit the CSS of the Events page schedule to match the Figma file, so that the HfLA meeting dates and times will be more legible for viewers.

This involves data pulled from VRMS, and may be difficult to accomplish

### Action Items
- [x] View the instructions/notes for editing the Events page schedule in the [Events page figma file, in a purple rectangle](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=62%3A94)
- [x] Find a way to separate the VRMS schedule data into two columns following the Figma design edits

### Resources/Instructions
[Events Page](https://www.hackforla.org/events)
[Figma- Events Page edit notes](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=62%3A94) (find detailed notes in purple rectangle, on bottom right of file)

The view on the left is existing output, the view on the right is expected output

image

 image

#1041

### PR DESCRIPTION
**Pull request by akibrhast**
This pull request is in relation to #1041 and it

closes #1109 
closes #1114 
closes #1112 
closes #1113 
closes #1111 
closes #1116 

😄 😄

### CODE REVIEW

**Comment by akibrhast** [comment]
Hey @jbubar @drubgrubby . Please take a moment to review this pull request
> the page looks good but I have some important edits and haven’t had time to list them out. Also a dev should be added as a reviewer as well, because splitting the page into separate html files is not that common (except for the home page, which was made a long time ago) so I just want to make sure that’s all fine (and they can verify code-related elements that I’m not knowledgable about). But first I just need to get my edits down, and I’m slammed today. I should be able to soon though. - @daniellex0 

Please await @daniellex0 styling edits, 
> so that you can focus on the code!

**Review by drubgrubby — COMMENTED** [review]
@akibrhast - Really impressive work.  You really took a lot of different issues on the events page, saw a different way to look at the issue, and then took it apart and put it back together in a new way that solves a lot of the issues.  

I am curious that you deconstructed the page and created all of these files to include, but left the javaScript on the .html page rather than moving it to a .js file and including it. Is there a reason that you left it here? 

That said, it's a different approach to structuring the site and building the pages, and even though I like it personally, I think it needs a discussion about if we want to go in this direction.  

@cnk @ExperimentsInHonesty - When you have a sec, can you weigh in on Akib's restructuring of the way the events page is built? It seems to work, but it's also a change from how we have been doing things, so a discussion seems warranted. 

I'm not looking at any of the design questions.  I will leave that for Danielle.

**Review by daniellex0 — CHANGES_REQUESTED** [review]
Ok @akibrhast sorry about the delay- thank you for making these changes! Some styling edits before this can go live (I think some of these issues are from a previous update to the page, but I appreciate if you can fix these to make the page look good for now before the other issues are done!):

1. Can you please actually remove the `content-section` class? It's causing the yellow banner to have 16px margin on both sides, which makes it look especially weird in mobile-

Screen Shot 2021-02-25 at 8 58 50 PM

I'll make this into a separate issue so that someone can figure it out after, because there seem to be some potential [solutions](https://css-tricks.com/full-width-containers-limited-width-parents/) (or we might just have to forego this standard class on this page.. not sure).

2. Please align the images with the text in both left and right column cards- 

So instead of this-

Screen Shot 2021-02-25 at 8 44 54 PM

it should look closer to this-

Screen Shot 2021-02-25 at 10 04 18 PM

I believe this can be done by putting the text and image in the same div, and their new class should have the following properties (**only for above mobile size**):

```
display: flex;
align-items: center;
```

3. In `.notification-banner p` class please remove the `line-height` property (it should go to default line height)

4. Please make these highlighted changes to the collapsed column cards:

Screen Shot 2021-02-25 at 10 22 28 PM

5. Please remove these lines:

Screen Shot 2021-02-25 at 10 31 04 PM

6. And this is my bad, I made this edit to the text on Figma but I'm realizing it actually looks too big, so please change the font size here back to 1rem (but keep the html text in all caps) and remove the line height, as seen here:

Screen Shot 2021-02-25 at 10 39 02 PM


That should be good for now, I'll make other edits on the other pages. Please just make sure everything (including my edits here) look fine in mobile before it goes live :)

**Comment by akibrhast** [comment]
@daniellex0 Made all the changes as request!

image

image

**Review by daniellex0 — CHANGES_REQUESTED** [review]
Thank you for making all of the changes @akibrhast ! Ok, truly, last changes *I promise* (any others will go in the other issues). Sorry, there are just some margin issues that I can't let go live on the site!

Just make these changes to these 4 classes please and you'll be done:

Screen Shot 2021-02-26 at 12 29 34 AM

margin-bottom instead of margin-top
Screen Shot 2021-02-26 at 12 22 00 AM

Screen Shot 2021-02-26 at 12 21 30 AM

Thank you!!

**Comment by akibrhast** [comment]
@daniellex0 best ever front end issue i had to deal with.... you tell me exactly what and where needs to be changed ... I go in an change it! 👯‍♂️  👯‍♂️
_Thankfully you are nice enough to take of playing with the css in dev tools and figuring out what works!!!_

File: events/our-locations-content.html [hunk 1]
```diff
@@ -0,0 +1,177 @@
+
+<div class="getting-started-page-card-locations">
+    <div class="locations-1">
+      <div class="location-img-pos">
+        <img class="location-image" src="../assets/images/hack-nights/westside1.png" alt="westside" />
+      </div>
+      <div class="location-content">
+        <h1 style="font-size: 35px; font-weight: normal">Monday</h1>
```
Comment (jbubar) [inline events/our-locations-content.html:8]: I dont think this was your doing. but there is inline styling here, that prevents the proper styling from showing up.

**Review by daniellex0 — APPROVED** [review]

**Comment by daniellex0** [comment]
Styling approved 👍

File: events/our-locations-content.html (hunk 1 above)
Comment (jbubar) [reply events/our-locations-content.html:8]: Could you delete these inline styles and style it in ths scss?
something like 
```
.location-content h1{
    font-size: 1.75rem;
    font-weight: 700;
}
```
got that from the [figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=6777%3A16)

File: events/our-locations-content.html [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+    </div>
+  </div>
+
+  <div class="getting-started-page-card-locations">
+    <div class="locations-1">
+      <div class="location-img-pos">
+        <img class="location-image" src="../assets/images/hack-nights/DTLA.png" alt="DTLA" />
+      </div>
+      <div class="location-content">
+        <h1 style="font-size: 35px; font-weight: normal">Tuesday</h1>
```
Comment (jbubar) [inline events/our-locations-content.html:90]: also here

File: events/our-locations-content.html [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+    </div>
+  </div>
+
+  <div class="getting-started-page-card-locations">
+    <div class="locations-1">
+      <div class="location-img-pos">
+        <img class="location-image" src="../assets/images/hack-nights/southla.png" alt="southLA" />
+      </div>
+      <div class="location-content">
+        <h1 style="font-size: 35px; font-weight: normal">Wednesday</h1>
```
Comment (jbubar) [inline events/our-locations-content.html:158]: and here

**Review by jbubar — CHANGES_REQUESTED** [review]
Really really cool stuff. I always learn from reading your code..

A few things that I noticed that were not a part of this issue...
maybe we should have these be scrollable?
Screen Shot 2021-02-26 at 9 54 11 AM

and  this does not look so good in mobile view
Screen Shot 2021-02-26 at 9 57 43 AM

But I know that teddy already created an issue for that.

I will finish my review later.. @akibrhast just informed me that our live wins page is down

**Comment by akibrhast** [comment]
@jbubar I have already created a separate issue that address the change you requested . There is already a LOOOOT of changes on this one pull request(Not proud of it and I am sorry to whoever is having to review this). Should we add more into it?
https://github.com/hackforla/website/issues/1115

---

The our-locations-content.html is a repeatable pattern that needs to be in a for loop somehow. That is why I did not address any of the changes requested in figma for that particular section. I have spoken about this to @daniellex0  and she has agreed on separating it into a separate issue. The same goes for the crappy mobile display, which @TeddyTelanoff made an issue for.
- #1115 
- #1110
- #1106

I would be very averse to doing this

image

I fear that if I make the change now, the bigger problem of having three hard coded divs, which themselves has 6 to 8 hard coded divs will be kind of forgotten!

thoughts? opinions?

**Review by jbubar — APPROVED** [review]
Thanks Akib for clarifying! and thanks for creating all of those issues!
Everything looks great! I like the new structure!

**Comment by akibrhast** [comment]
<details>
<summary>This is what the page looks like after all the changes applied in desktop</summary>

image

</details>
<details>
<summary>This is what the page looks like after all the changes applied in pixel2XL</summary>

image

</details>

**Comment by jbubar** [comment]
Here is a similar restructure with the wins page #1132

**Comment by akibrhast** [comment]
> @ExperimentsInHonesty - For now we are going to leave  it so such that a new page follows the directory structure as in this pull request.
 


A new issue should be created before merging this, that should be put into icebox and made into a discussion regarding the directory structure going forward at hack4la

**Comment by jbubar** [comment]
#1138

**Pull request merged**